RadixWallet/Clients/DeviceFactorSourceClient/DeviceFactorSourceClient+Live.swift

```diff
@@ -20,7 +21,7 @@ extension DeviceFactorSourceClient: DependencyKey {
 				// FIXME: Uh this aint pretty... but we are short on time.
 				if let overridingSnapshot = maybeSnapshot {
```

Comment:
I'm not a fan of naming optionals `maybe-`, to me it is clear from context that's it's optional, but i guess it's a matter of taste

Reply:
not important enough to mess with my `number of commits: 123, number of changed files: 123` miracle!